src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-infra-hosts.mdx

```diff
@@ -0,0 +1,308 @@
+          ```
+          resource:
+              attributes:
+                - key: host.id
+                  from_attribute: host.name
+                  action: upsert
+          ```
+
+      </td>
+    </tr>
+    <tr>
+      <td>
+        `processors::batch`
+      </td>
+      <td>
+        The [batch processor](https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/batchprocessor/README.md) accepts spans, metrics, or logs and places them into batches. Batching helps better compress the data and reduce the number of outgoing connections required to transmit the data. This processor supports both size and time based batching.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        `processors::memory_limited`
+      </td>
+      <td>
+        The [memory limiter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/memorylimiterprocessor) processor is used to prevent out of memory situations on the collector. Given that the amount and type of data the collector processes is environment specific and resource utilization of the collector is also dependent on the configured processors, it is important to put checks in place regarding memory usage.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        `processors::cumulative_delta`
+      </td>
+      <td>
+        The [cumulative delta](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/cumulativetodeltaprocessor) processor converts cumulative sum metrics to cumulative delta which is recommended in order to query system metric rates more easily in New Relic One.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        `service::pipelines::metrics`
+      </td>
+      <td>
+        Make sure `hostreceiver` and `resourcedetection` are included.
+      </td>
+    </tr>
+    </tbody>
+</table>
+
+Here is a sample configuration YAML file:
+
+```
+extensions:
+  health_check:
+
+receivers:
+  hostmetrics:
+    collection_interval: 20s
+    scrapers:
+      cpu:
+        metrics:
+          system.cpu.utilization:
+            enabled: true
+      load:
+      memory:
+        metrics:
+          system.memory.utilization:
+            enabled: true
+      disk:
+      filesystem:
+        metrics:
+          system.filesystem.utilization:
+            enabled: true
+      network:
+      paging:
+        metrics:
+          system.paging.utilization:
+            enabled: true
+      processes:
+      process:
+
+processors:
+  memory_limiter:
+    check_interval: 1s
+    limit_mib: 1000
+    spike_limit_mib: 200
+  batch:
+  cumulativetodelta:
+    metrics:
+      - system.network.io
+      - system.disk.operations
+      - system.network.dropped
+      - system.network.packets
+      - process.cpu.time
+  resource:
+    attributes:
+      - key: host.id
+        from_attribute: host.name
+        action: upsert
+  resourcedetection:
+    detectors: [ env, system]
```

Comment:
```suggestion
    detectors: [env, system]
```

Reply:
@josemore, I removed the space. GitHub seems to be having some problems today, so I wasn't able to accept your suggestion.